### ISSUE TITLE
Deprecate twisted.protocols.mice

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8148]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8148|
|Type|task|
|Created|2015-12-18 15:34:02Z|
|Branch|https://github.com/twisted/twisted/tree/why-do-we-even-have-a-mouse-protocol-8148|

The only implementation in there is for the "Logitech MouseMan serial mouse protocol", using a port that most computers don't even have anymore. We should deprecate it.

<details><summary>Searchable metadata</summary>

```
trac-id__8148 8148
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_why_do_we_even_have_a_mouse_protocol_8148 branches/why-do-we-even-have-a-mouse-protocol-8148
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1450452842860256 1450452842860256
changetime__1452341007787613 1452341007787613
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [46527]) Branching to why-do-we-even-have-a-mouse-protocol-8148.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Whoops, forgot to put this up for review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Look good. Thanks. Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [46608]) Merge why-do-we-even-have-a-mouse-protocol-8148: Deprecate twisted.protocols.mice

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8148](https://github.com/twisted/twisted/issues/8148)
